danieltichiyama/exercise-app
# Pwa setup

PWA initial setup complete, minimal functionality.

To test:
1.  docker-compose down -v && docker-compose up,
2.  npm run build,
3.  in an INCOGNITO tab, go to localhost:8080,
4.  Page loads normally, all app links should take you to the registration page,
5.  Login with calmoon808@gmail.com,
6.  App should automatically take you to the page you were on (needs to be fixed)
7.  open dev tools, Application, Service Worker, select 'Offline',
8.  all pages should reload but no data will be populated (this is the minimal part...)

optional:

1.  download and setup android studio, set up an AVD emulator,
2.  open site in chrome
3.  click on three dots and click Add website to home screen
4.  should show basic fitworks logo and the app should be available on home screen
5.  opening on home screen will open a splash screen followed by the app itself
6.  app should not have an address bar or other chrome browser window functions visible.
